assets/schema_input.json

```diff
@@ -25,6 +25,13 @@
                 "pattern": "^\\S+\\.mlst(\\.subtyping)?\\.json(\\.gz)?$",
                 "errorMessage": "MLST JSON file from locidex report, cannot contain spaces and must have the extension: '.mlst.json', '.mlst.json.gz', '.mlst.subtyping.json', or 'mlst.subtyping.json.gz'"
             },
+            "fastmatch_category": {
```

Comment:
I think the specific behaviour of this column may need a bit further discussion later on, but is something we can leave for this PR (and likely for this sprint to get feedback from others).

Specifically, on the nextflow side, the data in this column is being moved into the `meta` object for each sample. However, we cannot use the keyword `"meta"` in this schema JSON file, since that is used by IRIDA Next to load data from the metadata table in IRIDA Next.

I think it would make most sense to actually use the `"meta"` keyword in this JSON file, but maybe change the behaviour of IRIDA Next somehow? Or, to allow loading of a metadata column OR user-entered values to set query/reference samples.

However, as this is a more complex use case it requires further discussion. So this is good as-is now. I just wanted to make a note here about this.